### ISSUE TITLE
power cpu

### ISSUE THREAD

**Issue by Tannyll** [xmrig/xmrig#2317]
Hi,

Does it work with IBM power cpu?

**Comment by Spudz76** [comment]
Nope.  Mainly because no devs have one.  See also #2227 (there is some demand for the feature, but that doesn't change the fact we have no way to develop on one in-situ and tossing guesses out for people to try is far too slow of a dev loop).
